components/apps_form/apps_form_field/apps_form_field.tsx

```diff
@@ -21,6 +21,8 @@ import Provider from 'components/suggestion/provider';
 
 import Markdown from 'components/markdown';
 
+import {UserAutocomplete} from '@mattermost/types/autocomplete';
```

Comment:
rearrange this to after `import {UserProfile} from '@mattermost/types/users';`
So similar imports can be arranged closer to each other for clarity